core/tracker/tracker.go

```diff
@@ -86,12 +89,13 @@ type Tracker struct {
 }
 
 // New returns a new Tracker.
```

Comment:
// New returns a new Tracker. The deleter deadliner must return well after analyser deadliner since duties of the slot are often analysed together.